doc/stdenv/stdenv.chapter.md

```diff
@@ -116,6 +116,66 @@ On Linux, `stdenv` also includes the `patchelf` utility.
 
 ## Specifying dependencies {#ssec-stdenv-dependencies}
 
+Build systems often require more dependencies than just what `stdenv` provide. This section describes attributes that `stdenv.mkDerivation` accepts and which can be used to make these dependencies available to the build system.
+
+### Simplified overview
```

Comment:
Please add an anchor so that we have a stable reference we can link.

```suggestion
### Simplified overview {#ssec-stdenv-dependencies-overview}
```